pkgs/applications/editors/hmm/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, lib
+, makeWrapper
+, php
+, ncurses
+, coreutils
+, xclip
+}:
+
+
+stdenv.mkDerivation rec {
+  name = "h-m-m";
+
+  src = builtins.fetchGit {
+    url = "https://github.com/nadrad/h-m-m.git";
+    rev = "ecf74433844e5542f78f64f8a79807818743adff";
+  };
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r h-m-m $out/bin/h-m-m
+  '';
+  postFixup = ''
+    wrapProgram $out/bin/h-m-m \
+      --set PATH ${lib.makeBinPath [
+        php
+        ncurses
+        coreutils
+        xclip
+      ]}
+  '';
+
+  meta = with lib; {
+    description = "h-m-m is a simple, fast, keyboard-centric terminal-based tool for working with mind maps.";
+    homepage = "https://github.com/nadrad/h-m-m";
+    license = licenses.gpl3Only;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ tartavull ];
+  };
+}
```

Comment:
You probably need to add this package to pkgs/top-level/all-packages.nix as well, I don't think I see it in this PR.